src/styles/_grid.scss

```diff
@@ -1,85 +1,85 @@
-.col {
-  &-1 {
-    width: 8.33%;
-  }
-  &-2 {
-    width: 16.66%;
-  }
-  &-3 {
-    width: 25%;
-  }
-  &-4 {
-    width: 33.33%;
-  }
-  &-5 {
-    width: 41.66%;
-  }
-  &-6 {
-    width: 50%;
-  }
-  &-7 {
-    width: 58.33%;
-  }
-  &-8 {
-    width: 66.66%;
-  }
-  &-9 {
-    width: 75%;
-  }
-  &-10 {
-    width: 83.33%;
-  }
-  &-11 {
-    width: 91.66%;
-  }
-  &-12 {
-    width: 100%;
-  }
-}
-
-@media screen and (max-width: $media-tablet) {
-  .col {
-    &-1 {
-      width: 16.66%;
-    }
-    &-2 {
-      width: 33.33%;
-    }
-    &-3 {
-      width: 50%;
-    }
-    &-4 {
-      width: 66.66%;
-    }
-    &-5 {
-      width: 83.33%;
-    }
-    &-6,
-    &-7,
-    &-8,
-    &-9,
-    &-10,
-    &-11,
-    &-12 {
-      width: 100%;
-    }
-  }
-}
-
-@media screen and (max-width: $media-mobile) {
-  .col {
-    &-1 {
-      width: 50%;
-    }
-    &-2,
-    &-3,
-    &-4,
-    &-5 {
-      width: 100%;
-    }
-  }
-}
-.row {
-  display: flex;
-  flex-wrap: wrap;
-}
+.col {

```

Comment:
I suppose this file has already removed

Reply:
this class was created before, so I'm not sure if I can remove it